CONTRIBUTING.md

```diff
@@ -127,35 +271,37 @@ The development cycle for qiskit-aqua is informed by release plans in the
  
 ### Branches
 
-* `master`
+* `master`:
 
-The master branch is used for development of the next version of qiskit-aqua.
+The master branch is used for development of the next version of qiskit-ignis.
```

Comment:
Ignis should be  Aqua

Reply:
Fixed in: https://github.com/Qiskit/qiskit-aqua/pull/1002/commits/656d12c0b86f93fc0490b749f56c52bce841068d